Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -292,6 +292,9 @@ def create_incident(notable_data, occurred):
 
         incident["occurred"] = occurred
         notable_data = parse_notable(notable_data)
+        notable_id = notable_data.get('rule_id')
```

Comment:
Why is this called a notable ID? can you explain?

Reply:
It’s the notable id on the Splunk side.